diffsims/crystallography/get_hkl.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2023 The diffsims developers
+#
+# This file is part of diffsims.
+#
+# diffsims is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# diffsims is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with diffsims.  If not, see <http://www.gnu.org/licenses/>.
+
+from itertools import product
+
+import numpy as np
+from orix.vector import Vector3d
+
+
+def get_highest_hkl(lattice, min_dspacing=0.5):
+    """Return the highest Miller indices hkl of the plane with a direct
+    space interplanar spacing greater than but closest to a lower
+    threshold.
+
+    Parameters
```

Comment:
```suggestion
    """Return the highest Miller indices hkl of the plane with a direct
    space interplanar spacing(d-spacing) greater than but closest to the ``min_dspacing``.

    Parameters
```

Reply:
Thanks, I've added "(d-spacing)" and referred to the variable name (in italics instead of code-style, as used by NumPy).